autospec/pkg_integrity.py

```diff
@@ -210,32 +206,43 @@ def __del__(self):
         print(SEPT)
 
 
-def head_request(url):
-    curl = pycurl.Curl()
-    curl.setopt(curl.URL, url)
-    curl.setopt(curl.CUSTOMREQUEST, "HEAD")
-    curl.setopt(curl.NOBODY, True)
-    curl.setopt(curl.TIMEOUT, 5)
-    curl.setopt(curl.FOLLOWLOCATION, True)
-    curl.perform()
-    http_code = curl.getinfo(pycurl.HTTP_CODE)
-    curl.close()
-    return http_code
+def download_file(url, destination):
+    sign_file = os.path.join(destination, os.path.basename(url))
+    print("Attempting {}".format(sign_file))
+    with open(sign_file, 'wb') as f:
+        curl = pycurl.Curl()
+        curl.setopt(curl.URL, url)
+        curl.setopt(curl.WRITEDATA, f)
+        curl.setopt(curl.FOLLOWLOCATION, True)
```

Comment:
Just a suggestion: Perhaps set `curl.FAILONERROR` too, so that http codes > 400 will automatically result in an error? Because the previous code didn't do this, it's not a huge deal right now.